docs/tutorial.md

```diff
@@ -783,3 +791,25 @@ pod/memcached-sample-6c765df685-mfqnz                      1/1     Running   0
 
 If you modify the size field of the `memcached-sample.yaml` and re-apply it. The
 operator will trigger a reconcile and adjust the sample pods to the size given.
+
+### Deploy your Operator with OLM
+First, install [OLM](https://sdk.operatorframework.io/docs/olm-integration/tutorial-bundle/#enabling-olm):
+
+```
+operator-sdk olm install
+```
+
+Bundle your operator, then build and push the bundle image. The [bundle](https://github.com/operator-framework/operator-registry/blob/v1.16.1/docs/design/operator-bundle.md#operator-bundle) target generates a bundle in the bundle directory containing manifests and metadata defining your operator. bundle-build and bundle-push build and push a bundle image defined by bundle.Dockerfile.
```

Comment:
Should we reference the latest release bundle documentation?

```suggestion
Bundle your operator, then build and push the bundle image. The [bundle](https://github.com/operator-framework/operator-registry/blob/v1.23.0/docs/design/operator-bundle.md#operator-bundle) target generates a bundle in the bundle directory containing manifests and metadata defining your operator. bundle-build and bundle-push build and push a bundle image defined by bundle.Dockerfile.
```

Reply:
Yes right. updated it.